spec/controllers/api/v2/searches_controller_spec.rb

```diff
@@ -26,6 +26,11 @@
       filter: '2'
     }
   end
+  let(:routed_query_setup) do
+    routed_query = affiliate.routed_queries.build(url: "http://www.gov.gov/foo.html", description: "testing")
+    routed_query.routed_query_keywords.build(keyword: 'foo bar')
+    routed_query.save!
+  end
```

Comment:
You could skip this setup by using the `basic_affiliate` for the routed queries specs. That affiliated is already associated w/ routed query fixtures:
```
spec/fixtures/routed_queries.yml

unclaimed_money:
  affiliate: :basic_affiliate
  url: "https://www.usa.gov/unclaimed_money"
  description: Everybody wants it
moar_unclaimed_money:
  affiliate: :basic_affiliate
  url: "https://www.usa.gov/unclaimed_money"
  description: Seriously they do
```
You'd need to update the spec setup & expectations to match the fixtures, but it would clean up the spec.